packages/babel-plugin-transform-class-properties/README.md

```diff
@@ -76,11 +76,11 @@ require("babel-core").transform("code", {
 
 ## Options
 
-### `spec`
+### `loose`
 
 `boolean`, defaults to `false`.
 
-Class properties are compiled to use `Object.defineProperty`. Static fields are now defined even if they are not initialized.
+Class properties are compiled use an assignment expression instead of `Object.defineProperty`. Static fields are now not defined if they are not initialized.
```

Comment:
i think it was never added for the default mode (which is now changing to loose)